src/main/java/github/scarsz/discordsrv/commands/CommandResync.java

```diff
@@ -14,6 +15,10 @@
             permission = "discordsrv.resync"
     )
     public static void execute(CommandSender sender, String[] args) {
+        if (DiscordSRV.config().getMap("GroupRoleSynchronizationGroupsAndRolesToSync").isEmpty()) {
```

Comment:
Would be better to also check that the ids aren't/are valid incase the option is on default values

Reply:
Included a check on default and a check for roleId. Feel free to pick which you'd prefer (as both seems unnecessary).